qutip/core/cy/cqobjevo.pxd

```diff
@@ -34,29 +34,29 @@
 
 cimport numpy as cnp
 
-from qutip.core.cy.cqobjevo_factor cimport CoeffFunc
 from qutip.core.data.base cimport idxint
 from qutip.core.data cimport CSR, Dense, Data
 
 cdef class CQobjEvo:
-    cdef readonly (idxint, idxint) shape
+    cdef readonly (idxint, idxint) _shape
```

Comment:
How come you made this private?  I access this in Python-space (I think - I may have removed it again), so shouldn't it be public?  Same question for `_issuper` below.

Reply:
I thought you made it private, it became like this after merging some changed in your branch...
I will change it.